### DISCUSSION TITLE
user-guide/import-from-other-software

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#249] [文档评论区]
# user-guide/import-from-other-software

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/import-from-other-software

**Comment by TJYYWXBG** [comment]
谢谢贡献者的教程与维护。
建议第一步(找不到enl库文件的操作说明)放到后面去，作为补充说明即可。放在最开始有一定的干扰性。

**Reply by northword** [reply]
我们在 https://github.com/zotero-chinese/wiki/pull/250 对这个问题做出了修改，你可以点击链接里的 “Files Changed” 选项卡 或是 链接里第一个评论里的“预览地址” 来审查更改，欢迎你提出意见和建议。
